common/src/main/java/com/brandwatch/minibcr/common/repository/MentionRepositoryImpl.java

```diff
@@ -0,0 +1,84 @@
+package com.brandwatch.minibcr.common.repository;
+
+import com.brandwatch.minibcr.common.domain.Mention;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.dao.EmptyResultDataAccessException;
+import org.springframework.jdbc.core.namedparam.MapSqlParameterSource;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.jdbc.core.support.JdbcDaoSupport;
+import org.springframework.stereotype.Repository;
+
+import javax.annotation.PostConstruct;
+import javax.sql.DataSource;
+import java.util.List;
+import java.util.Map;
+
+import com.google.common.collect.ImmutableMap;
+
+@Repository
+public class MentionRepositoryImpl extends JdbcDaoSupport implements MentionRepository {
```

Comment:
Why did you decide to use JdbcDaoSupport?

Reply:
It helped with getting a Jdbc template using `getJdbcTemplate()` without having to create a new one. However it looks unnecessary so I'll remove it